### PR TITLE
Bump Version of Grpc.Net.Client

### PR DESCRIPTION
**Pull request by chlafreniere**
Dependency of Microsoft.Azure.Functions.Worker.

Grpc.Net.Client used in tests and samples only.

### CODE REVIEW

File: samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Timer" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues" />
     <PackageReference Include="Microsoft.AspNetCore.Http" />
+    <PackageReference Include="Grpc.Net.Client" VersionOverride="2.55.0" />
```
Comment (Charles-Gagnon) [inline samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj:14]: Do you need to use VersionOverride? You should just be able to add it to https://github.com/Azure/azure-functions-sql-extension/blob/main/Directory.Packages.props and then reference it directly here

Comment (chlafreniere) [reply samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj:14]: Yes I believe I do and I tried doing what you mentioned first; please take a look at this branch where I tried doing that: https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/directorypacks?expand=1

https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/directorypacks?expand=1#diff-2b567884a3a252859000ac56f9a9caed8eab3d1ce97addce0cc4996a7f43ee02R839

Comment (Charles-Gagnon) [reply samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj:14]: you still need to add the package reference in the two csproj's. Once you do that it should pick up the specified version

Comment (Charles-Gagnon) [reply samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj:14]: (you just won't specify the version when doing so)

Comment (chlafreniere) [reply samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj:14]: ahhh that's why i was so confused. ok on it!

**Review by Charles-Gagnon — APPROVED** [review]

**Review by VasuBhog — APPROVED** [review]

**Review by MaddyDev — APPROVED** [review]

**Review by lucyzhang929 — APPROVED** [review]

**Pull request merged**